homeassistant/components/fitbit/sensor.py

```diff
@@ -345,6 +348,7 @@ class FitbitSensor(SensorEntity):
     def __init__(
         self,
         client: Fitbit,
+        user_profile: dict,
```

Comment:
I assume all dict keys are string:
```suggestion
        user_profile: dict[str, Any],
```

Reply:
Updated, thanks!